custom_components/tahoma/water_heater_devices/domestic_hot_water_production.py

```diff
@@ -33,6 +39,22 @@ class DomesticHotWaterProduction(OverkizEntity, WaterHeaterEntity):
     )
     _attr_temperature_unit = TEMP_CELSIUS
 
+    @property
+    def _is_boost_mode_on(self):
```

Comment:
Here also we can return nothin,  while True/False is expected. Can you also please type the method?

Reply:
@rguillard77 I realize you didn't add anything for `DHWP_TYPE_MBL` regarding boost mode. Do you have this mode in your device?